extension/src/experiments/columns/extract.ts

```diff
@@ -88,6 +89,7 @@ export const extractColumns = (
   ].join('\n')
 
   const columns: Columns = {
+    Timestamp: experiment?.timestamp,
```

Comment:
Is there a reason for using the capitalized `Timestamp`?

Reply:
Using a capitalized Timestamp helps us to write less code since that's what the label is referred to as :)

Reply:
I think now would be a good time to rename this to `Created` to match the CLI table:

<img width="544" alt="image" src="https://user-images.githubusercontent.com/37993418/186774833-bf153c6b-0119-4bd5-9443-9ecfa0e7d0fb.png">

Can do it in follow up if necessary.